.drone.yml

```diff
@@ -0,0 +1,469 @@
+    <<: *sonar_scanner
+    when:
+      branch:
+        include:
+          - master
+      event: push
+
+  - name: build_image
+    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+    commands:
+      - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
+      - docker build --no-cache -t $${IMAGE_REPO}:$${DRONE_COMMIT_SHA} .
+    volumes:
+      - name: dockersock
+        path: /var/run
+    when:
+      branch:
+        include:
+          - master
+          - feature/*
+      event: [push, pull_request]
+
+  - name: image_to_ecr
+    image: plugins/ecr
+    settings:
+      access_key:
+        from_secret: aws_access_key_id
+      secret_key:
+        from_secret: aws_secret_access_key
+      region: eu-west-2
+      repo: sas/fmr
+      registry: 340268328991.dkr.ecr.eu-west-2.amazonaws.com
+      tags:
+        - ${DRONE_COMMIT_SHA}
+    when:
+      branch: 
+        include: 
+          - master
+          - feature/*
+      event: [push, pull_request]
+
+  # Trivy Security Scannner for scanning nodejs packages in Yarn
+  - name: scan_node_packages
+    pull: always
+    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
+    resources:
+      limits:
+        cpu: 1000
+        memory: 1024Mi
+    environment:
+      IMAGE_NAME: sas/fmr:${DRONE_COMMIT_SHA}
+      SEVERITY: MEDIUM,HIGH,CRITICAL  --dependency-tree --format table
+      FAIL_ON_DETECTION: false
+      IGNORE_UNFIXED: false
+      ALLOW_CVE_LIST_FILE: hof-services-config/E_Visa_Find_My_Reference_Form/trivy/.trivyignore.yaml
+    when:
+      event: [push, pull_request]
+
+  # Deploy to pull request UAT environment
+  - name: deploy_to_branch
+    pull: if-not-exists
+    image: quay.io/ukhomeofficedigital/kd:v1.14.0
+    environment:
+      # NOTIFY_STUB: stub
+      KUBE_SERVER:
+        from_secret: kube_server_dev
+      KUBE_TOKEN:
+        from_secret: kube_token_dev
+    commands:
+      - bin/deploy.sh $${BRANCH_ENV}
+    when:
+      branch:
+        include:
+          - master
+          - feature/*
+      event: pull_request
+
+  - name: sonar_scanner_branch
+    <<: *sonar_scanner
+    when:
+      branch:
+        include:
+          - master
+          - feature/*
+      event: pull_request
+
+  - name: deploy_to_uat
+    pull: if-not-exists
+    image: quay.io/ukhomeofficedigital/kd:v1.14.0
+    environment:
+      KUBE_SERVER:
+        from_secret: kube_server_dev
+      KUBE_TOKEN:
+        from_secret: kube_token_dev
+    commands:
+      - sh bin/deploy.sh $${UAT_ENV}
+    when:
+      branch:
+        include:
+          - master
+      event: push
+
+  # - name: acceptance_tests_uat
+  #   <<: *acceptance_tests
+  #   commands:
+  #     - export ACCEPTANCE_HOST_NAME=https://{{APP_NAME}}.internal.uat.sas-notprod.homeoffice.gov.uk
+  #     - npx playwright install
+  #     - npm run test:acceptance
+  #   when:
+  #     branch: master
+  #     event: push
+
```

Comment:
this can be removed, there are no plans for an acceptance test in this context

Reply:
This is removed